ir_rule_restriction/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,14 @@
+Right now, Odoo defines to types of record rules:
+Global: All of them must have allow the action to be performed in order for
+the operation a user is performing to succeed. Therefore, an AND is
+performed in all of them.
+Group-specific: Any of them can allow the action to be performed in order for
+the operation a user is performing to succeed. Therefore, an OR is
+performed in all of them.
+
+Which means that if we have userA of groupA which global rules allow them to
```

Comment:
To have this effect it might be a better idea to use the concept of an exclusion rule. it means do NOT give the asked for permission to the records that apply, no matter if other rules would grant those permissions.